scanner/src/test/kotlin/scanners/scancode/ScanCodeResultParserTest.kt

```diff
@@ -73,6 +73,19 @@ class ScanCodeResultParserTest : WordSpec({
         }
     }
 
+    "ScanCode 30 results" should {
```

Comment:
This (and other test names) should probably consistently refer to either "30.1.0" or "30.*". Or maybe yet better, we should stop referring to the ScanCode version at all, and instead refer to the "output_format_version" written in the JSON, which current is at 1.0.0.

Reply:
> the old test assets don't have an "output_format_version" so I have kept them as is.

That's because the "output_format_version" JSON property was just recently introduced, see https://github.com/nexB/scancode-toolkit/issues/2653#issuecomment-924934429.